tests/unit/test_services.py

```diff
@@ -481,6 +481,14 @@ def test_fix_code_functions_emit_debug_logs(
             "Restoring jinja2 variables...",
             "Restoring double exclamations...",
             "Fixing comments...",
+            # Fixing comments causes a re-run of fixers, so we get duplicates from here
```

Comment:
I don't like the re-running of previous fixers, can't you fix comments before so that we don't have to rerun everything?

I think it is really bad from the point of view of the performance of the tool to do this.